lib/jwt/jwk/rsa.rb

```diff
@@ -51,6 +51,10 @@ def verify_key
       def export(options = {})
         exported = parameters.clone
         exported.reject! { |k, _| RSA_PRIVATE_KEY_ELEMENTS.include? k } unless private? && options[:include_private] == true
+
+        exported[:x5c] = Base64.strict_encode(rsa_key.to_der) if options[:x5c]
```

Comment:
A few concerns about this.

Is the x5c header really meant to contain just a key. Isn't that supposed to have the whole certificate chain?

The header should probably be an array.

I have a feeling that `#to_der` is including the private key if such exist, probably should be specific to the public key?

Reply:
I don't have a good idea about it because I encounter only token with `x5t` header. I think we can skip this part and revisit it later when it becomes necessary. What do you think?